BrainPortal/lib/act_rec_log.rb

```diff
@@ -272,7 +272,7 @@ def addlog_context(context, message=nil, caller_level=0)
     class_name     = context.class.to_s
     class_name     = context.to_s if class_name == "Class"
     rev_info       = context.revision_info
-    pretty_info    = rev_info.svn_id_rev
+    pretty_info    = rev_info.short_commit
```

Comment:
When I upload a file, the author is displayed as "(author?)" in the file log. I don't know where this comes from though, because neither the Userfile model nor the CSVFile class write this line to the log
```
[2016-08-15 14:08:10 EDT] CSVFile revision 6082312d (author?) 2015-10-29
[2016-08-15 14:08:10 EDT] create() UserfilesController rev. 918a790c Uploaded by admin
```